src/Components/Reusable/Staking/StargateLockedValue.tsx

```diff
@@ -38,6 +39,56 @@ export const StargateLockedValue = ({ isLoading, nfts = [], rootStyle }: Props)
     const fiatBalance = useMemo(() => {
         return BalanceUtils.getFiatBalance(totalLockedVet.toString, vetTokenInfo.exchangeRate ?? 1, VET.decimals)
     }, [totalLockedVet.toString, vetTokenInfo.exchangeRate])
+
+    const renderLockedValue = useMemo(() => {
+        if (isLoading) {
+            return (
+                <BaseSkeleton
+                    height={20}
+                    width={60}
+                    boneColor={theme.colors.skeletonBoneColor}
+                    highlightColor={theme.colors.skeletonHighlightColor}
+                />
+            )
+        }
+
+        if (!isNodeOwner) {
+            return (
+                <BaseView style={styles.tagContainer}>
+                    <BaseText typographyFont="smallCaptionMedium" color={theme.colors.stakedCard.tagText}>
+                        {LL.STARGATE_DELEGATEE_LABEL()}
+                    </BaseText>
+                </BaseView>
+            )
+        }
```

Comment:
I'd have put `if (!isNodeOwner && !isLoading)` to use the fiat balance skeleton for both